README.md

```diff
@@ -549,6 +549,26 @@ Here are the metrics settings:
     "vm.max_map_count" = "262144"
     ```
 
+#### Boot-related settings for bare metal variants
+
+Specifying either of the following settings will generate a kernel boot config file to be loaded on subsequent boots:
+* `settings.boot.kernel-parameters`: These allows additional kernel parameters to be specified to the kernel command line during boot.
+* `settings.boot.init-parameters`: These allows additional init parameters to be specified to init during boot.
+
+*Please note that boot settings only exist for bare metal variants only*
+
+Example user data for specifying boot settings:
+
+```toml
+[settings.boot]
+kernel.console = ["tty0", "ttyS1"]
```

Comment:
Let's add a more complicated example including baud rate, flow control, etc.for the serial console:
`ttyS1,115200n8`